### PR TITLE
Day ten

### PR DESCRIPTION
**Pull request by Talaspaev**

### CODE REVIEW

File: src/Components/Dialogs/dialogs.jsx [hunk 1]
```diff
@@ -1,21 +1,41 @@
 import React from 'react';
 
+import { actionCreater } from '../../redux/state';
+
+import ACTION_TYPES from '../../actionTypes';
```
Comment (Dmitry-White) [inline src/Components/Dialogs/dialogs.jsx:5]: Consider grouping this import with other imports of the same level ("../../")

File: src/Components/Dialogs/dialogs.jsx [hunk 2]
```diff
@@ -1,21 +1,41 @@
...
 
 import style from './dialogs.module.css';
 
-
 const Dialogs = (props) => {
+  const { dispatch } = props;
   const { dialogsData } = props.state.dialogsPages;
+
+  const onAddNewMessage = () => {
+    dispatch(actionCreater(ACTION_TYPES.ADD_NEW_MESSAGE, null));
```
Comment (Dmitry-White) [inline src/Components/Dialogs/dialogs.jsx:16]: There's a typo - "actionCreator", not "actionCreater".

File: src/Components/Dialogs/dialogs.jsx [hunk 3]
```diff
@@ -1,21 +1,41 @@
...
+    const text = e.target.value;
+    dispatch(actionCreater(ACTION_TYPES.UPDATE_NEW_MESSAGE, text));
+  };
+
   return (
+
     <div className={style.dialogs}>
+      <div className={style.textAreaWrapper}>
+        <textarea onChange={onChangeNewMessage} value={props.state.dialogsPages.newMessage} />
+        <button type="button" onClick={onAddNewMessage}>Button</button>
```
Comment (Dmitry-White) [inline src/Components/Dialogs/dialogs.jsx:29]: Consider moving the string "Button" that should be displayed on the button to I18N.

File: src/Components/Profile/MyPosts/myPosts.jsx [hunk 4]
```diff
@@ -1,23 +1,24 @@
 import React from 'react';
 
+import { actionCreater } from '../../../redux/state';
+
 import ACTION_TYPES from '../../../actionTypes';
 import Post from './Post/post';
 
 import style from './myPosts.module.css';
 
+
```
Comment (Dmitry-White) [inline src/Components/Profile/MyPosts/myPosts.jsx:10]: Consider removing this redundant empty line.

File: src/redux/state.js [hunk 5]
```diff
@@ -69,10 +88,18 @@ const store = {
...
+        this._updateNewMessage(action.payload);
+        break;
       default: this._renderFunc();
     }
   },
 
 };
 
-export default store;
+const actionCreater = (typeOf, text) => ({ type: typeOf, payload: text });
```
Comment (Dmitry-White) [inline src/redux/state.js:103]: Consider calling me for a live session regarding this functionality.

File: src/Components/Dialogs/dialogs.jsx (hunk 1 above)
Comment (Talaspaev) [reply src/Components/Dialogs/dialogs.jsx]: I groupted

File: src/Components/Dialogs/dialogs.jsx (hunk 3 above)
Comment (Talaspaev) [reply src/Components/Dialogs/dialogs.jsx]: Sure

File: src/Components/Profile/MyPosts/myPosts.jsx (hunk 4 above)
Comment (Talaspaev) [reply src/Components/Profile/MyPosts/myPosts.jsx]: Yeah.really redundant

File: src/redux/state.js (hunk 5 above)
Comment (Talaspaev) [reply src/redux/state.js]: At now actioncreator look like cool.thanks for call.It was usefull

**Pull request merged**